### PR TITLE
Flutter for iOS — ViewControllers section

### PR DESCRIPTION
**Pull request by rock3r**
This PR is for the fifth section of the work to create the `flutter-for-ios.md` document. Writing by @niamh-power on Novoda's behalf, I am just a proxy.

Following the same rough format as the `flutter-for-android.md` document, with iOS details instead.

The following sections have been created:

* What is the equivalent to ViewControllers in Flutter?
* How do I listen to iOS lifecycle events?

⚠️ **This requires #979 to be merged first!** ⚠️

### CODE REVIEW

File: flutter-for-ios.md [hunk 1]
```diff
@@ -27,3 +27,1085 @@ that are most relevant to your needs.
...
+In iOS, you can override methods to the `ViewController` to capture lifecycle
+methods for the view itself, or register lifecycle callbacks in the
+`AppDelegate`. In Flutter you have neither concept, but you can instead listen
+to lifecycle events by hooking into the `WidgetsBinding` observer and
+listening to the `didChangeAppLifecycleState()` change event.
+
+The lifecycle events you can observe are:
+
+* `inactive` — The application is in an inactive state and is not receiving
+user   input. This event only works on iOS, as there is no equivalent event to
```
Comment (mjohnsullivan) [inline flutter-for-ios.md]: Spacing

File: flutter-for-ios.md [hunk 2]
```diff
@@ -27,3 +27,1085 @@ that are most relevant to your needs.
...
+map to   on Android * `paused` — The application is not currently visible to
```
Comment (mjohnsullivan) [inline flutter-for-ios.md]: Spacing

**Comment by rock3r** [comment]
Contents covered in #993, closing this. Will address comments on that one.

File: flutter-for-ios.md (hunk 2 above)
Comment (rock3r) [reply flutter-for-ios.md]: Done on #993

File: flutter-for-ios.md (hunk 1 above)
Comment (rock3r) [reply flutter-for-ios.md]: Fixed on #993, not sure what happened there.

**Pull request closed without merging**